ecs-init/docker/docker.go

```diff
@@ -440,6 +459,22 @@ func (c *client) getHostConfig(envVarsFromFiles map[string]string) *godocker.Hos
 	return createHostConfig(binds)
 }
 
+// getCredentialsFetcherSocketBind returns the corresponding bind for credentials fetcher socket.
+func getCredentialsFetcherSocketBind() (string, bool) {
+	CredentialsFetcherUnixSocketHostPath := config.CredentialsFetcherUnixSocket()
```

Comment:
`config.HostCredentialsFetcherPath()` calls `CredentialsFetcherUnixSocket()` to get the unix socket host path right? can we use `config.HostCredentialsFetcherPath()` in all places in `docker.go` and `CredentialsFetcherUnixSocket` can be local to `config.go`?

Reply:
config.go is in ecs-init and docker.go is in ecs-agent code base, that is not possible

Reply:
i was referring to `ecs-init/config/`, this `CredentialsFetcherUnixSocket` resides in `ecs-init/config/common.go` isnt it?

Reply:
refactored the code accordingly